code/src/components/AlbumArtwork.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { AlbumActions } from './AlbumActions'
+
+export const AlbumArtwork = (props) => {
+  return (
+    <div className="artwork">
+      <AlbumActions />
+      <a href={props.albumUrl} target="_blank" rel="noreferrer">
+        <img className="artwork-image" alt={props.albumName} title={props.albumName} src={props.image} />
```

Comment:
There is a convention that says that if we pass more than 2 props to one component, we should format our code in a way that every props is written in separate line. In your code, it would look like this :

```
<img
  className="artwork-image"
  alt={props.albumName}
  title={props.albumName}
  src={props.image}
/>
```

We do this only for readability purpose, nonetheless it's important